### PR TITLE
RHDEVDOCS-3314 Add link from 5.0 RNs to RHBA-2021:81705-01 Openshift …

### PR DESCRIPTION
**Pull request by rolfedh**
…Logging Bug Fix Release (5.0.9)

- Aligned team: Dev Tools
- For branches: 4.7 only
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3314
- Direct link to doc preview: https://deploy-preview-36961--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-0-9
- SME review: @jcantrill 
- QE review: @kabirbhartiRH 
- Peer review: tbd
- <All reviews complete. Please merge now>

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
lgtm

File: modules/cluster-logging-release-notes-5.0.9.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[id="cluster-logging-release-notes-5-0-9"]
+= OpenShift Logging 5.0.9
+
+This release includes link:https://access.redhat.com/errata/RHBA-2021:3705[RHBA-2021:3705 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.9)].
```
Comment (Eva-Lotte) [inline modules/cluster-logging-release-notes-5.0.9.adoc:4]: maybe it's just me, but the link doesn't seem to work.

Comment (Eva-Lotte) [reply modules/cluster-logging-release-notes-5.0.9.adoc:4]: I am not sure how the part of the sentence before the link and in [] form a sentence.

File: modules/cluster-logging-release-notes-5.0.9.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+[id="openshift-logging-5-0-9-bug-fixes"]
+== Bug fixes
+
+This release also includes the following bug fixes:
```
Comment (Eva-Lotte) [inline modules/cluster-logging-release-notes-5.0.9.adoc:9]: I would remove "also".

File: modules/cluster-logging-release-notes-5.0.9.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+* Before this update, some log entries had unrecognized UTF-8 bytes, which caused Elasticsearch to reject the messages and block the entire buffered payload. This update resolves the issue: rejected payloads drop the invalid log entries and resubmit the remaining entries. (link:https://issues.redhat.com/browse/LOG-1574[LOG-1574])
```
Comment (Eva-Lotte) [inline modules/cluster-logging-release-notes-5.0.9.adoc:11]: which messages? If that's unspecified, remove "the" in front of messages?

File: modules/cluster-logging-release-notes-5.0.9.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+* Before this update, editing the `ClusterLogging` custom resource (CR) did not apply the value of the `total_limit_size` field. Instead, it applied the default value which represents maximum available size. With this update, Fluentd uses the value of the `total_limit_size` field or default value whichever is minimum. (link:https://issues.redhat.com/browse/LOG-1736[LOG-1736])
```
Comment (Eva-Lotte) [inline modules/cluster-logging-release-notes-5.0.9.adoc:13]: Can we mention what CR applied the value to? To Fluentd probably?

Comment (Eva-Lotte) [reply modules/cluster-logging-release-notes-5.0.9.adoc:13]: How about something that puts this more clear: "Fluentd uses the value of the `total_limit_size` field or the default value, depending on which one represents the minimum."
If you reject this suggestion, please at least add a comma before "whichever".

File: modules/cluster-logging-release-notes-5.0.9.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.9.adoc:4]: The broken link is expected. We wait for the advisory to be published before we merge the PR.
This line includes the advisory title and synopsis: `RHBA...` and `OpenShift...`. What you would recommend doing instead? I'd like to minimize the impact on other similar content.

Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.9.adoc:4]: https://deploy-preview-36961--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-0

File: modules/cluster-logging-release-notes-5.0.9.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.9.adoc:9]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.9.adoc (hunk 3 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.9.adoc:11]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.9.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.9.adoc]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.9.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.9.adoc:4]: I would prefer to keep the format of these links as they are. Altering them will involve making a lot of low-value changes to other files.

Comment (Eva-Lotte) [reply modules/cluster-logging-release-notes-5.0.9.adoc:4]: So basically you are saying that in the end the link will work? In that case I'd leave it as is!

**Comment by Eva-Lotte** [comment]
lgtm, very well done, as always!

**Pull request merged**